src/app/view-project/view-project.component.css

```diff
@@ -0,0 +1,27 @@
+h1 {
+  margin-left: 20em;
+}
+
+.inner-spacing {
+  padding: 0.5em;
```

Comment:
you can use p-1
read about bootstrap utility classes:
https://getbootstrap.com/docs/4.0/utilities/spacing/